internal/cache/cache.go

```diff
@@ -101,97 +100,20 @@ func (cc *CConfig) TTL() time.Duration {
 type Manager interface {
 	GetCache(cc *CConfig) (CInterface, error)
 	ResetCachesForNamespace(ns string)
-	ListKeys() []string
+	ListCacheNames(namespace string) []string
 }
 
-type CInterface interface {
-	Get(key string) interface{}
-	Set(key string, val interface{})
-
-	GetString(key string) string
-	SetString(key string, val string)
-
-	GetInt(key string) int
-	SetInt(key string, val int)
-
-	Delete(key string) bool
-}
-
-type CCache struct {
-	enabled   bool
-	ctx       context.Context
-	namespace string
-	name      string
-	cache     *ccache.Cache
-	cacheTTL  time.Duration
-}
-
-func (c *CCache) Set(key string, val interface{}) {
-	if !c.enabled {
-		return
-	}
-	c.cache.Set(c.name+":"+key, val, c.cacheTTL)
-}
-
-func (c *CCache) Get(key string) interface{} {
-	if !c.enabled {
-		return nil
-	}
-	if cached := c.cache.Get(c.name + ":" + key); cached != nil {
-		cached.Extend(c.cacheTTL)
-		return cached.Value()
-	}
-	return nil
-}
-
-func (c *CCache) SetString(key string, val string) {
-	c.Set(key, val)
-}
-
-func (c *CCache) GetString(key string) string {
-	val := c.Get(key)
-	if val != nil {
-		return c.Get(key).(string)
-	}
-	return ""
-}
-
-func (c *CCache) SetInt(key string, val int) {
-	c.Set(key, val)
-}
-
-func (c *CCache) GetInt(key string) int {
-	val := c.Get(key)
-	if val != nil {
-		return c.Get(key).(int)
-	}
-	return 0
-}
-
-func (c *CCache) Delete(key string) bool {
-	return c.cache.Delete(key)
-}
+type CInterface cache.CInterface
```

Comment:
now we just pass through the common cache interface. In the future, there will be a PR to clean this up